src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
-      }
-    },
-    "picture": "wilbert-rodriguez.webp"
+    "picture": "wilbert-rodriguez.webp",
+    "positions": { "S21": { "title": "Intern Program Manager", "tier": 22 } }
   },
   {
     "fullName": "Samuel Sandoval",
-    "positions": {
-      "S21": {
-        "title": "Vice President, Dev Director",
-        "tier": 1
-      }
-    },
-    "picture": "samuel-sandoval.webp"
+    "picture": "samuel-sandoval.webp",
+    "positions": { "S21": { "title": "Vice President, Dev Director", "tier": 1 } }
   },
   {
     "fullName": "Kavit Sanghavi",
-    "positions": {
-      "S21": {
-        "title": "Algo Director",
-        "tier": 11
-      }
-    },
-    "picture": "kavit-sanghavi.webp"
+    "picture": "kavit-sanghavi.webp",
+    "positions": { "S21": { "title": "Algo Director", "tier": 11 } }
   },
   {
     "fullName": "Parth Sharma",
-    "positions": {
-      "S21": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
-    "picture": "parth-sharma.webp"
+    "picture": "parth-sharma.webp",
+    "positions": { "S21": { "title": "Algo Officer", "tier": 12 } }
   },
   {
     "fullName": "Justin Stitt",
+    "picture": "justin-stitt.webp",
     "positions": {
-      "S22": {
-        "title": "Algo Officer",
-        "tier": 15
-      },
-      "F22": {
-        "title": "AI Team Lead",
-        "tier": 12
-      }
-    },
-    "picture": "justin-stitt.webp"
+      "S22": { "title": "Algo Officer", "tier": 12 },
+      "F22": { "title": "AI Team Lead", "tier": 11 }
+    }
   },
   {
     "fullName": "Johnson Tong",
-    "positions": {
-      "S21": {
-        "title": "Workshop Manager",
-        "tier": 6
-      }
-    },
-    "picture": "johnson-tong.webp"
+    "picture": "johnson-tong.webp",
+    "positions": { "S21": { "title": "Workshop Manager", "tier": 18 } }
   },
   {
     "fullName": "Alex Truong",
-    "positions": {
-      "F21": {
-        "title": "Algo Officer",
-        "tier": 15
-      }
-    },
+    "positions": { "F21": { "title": "Algo Officer", "tier": 12 } },
     "picture": "alex-truong.webp"
   },
   {
     "fullName": "Daniel Truong",
+    "picture": "daniel-truong.webp",
     "positions": {
-      "S22": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      },
-      "F22": {
-        "title": "Dev Project Manager",
-        "tier": 22
-      }
-    },
-    "picture": "daniel-truong.webp"
+      "S22": { "title": "Dev Project Manager", "tier": 18 },
+      "F22": { "title": "Dev Project Manager", "tier": 18 }
+    }
   },
   {
     "fullName": "Samuel Valls",
+    "picture": "samuel-valls.webp",
     "positions": {
-      "S21": {
-        "title": "Community Manager",
-        "tier": 10
-      },
-      "F21": {
-        "title": "Create Officer",
-        "tier": 17
-      },
-      "S22": {
-        "title": "Create Operations Manager",
-        "tier": 19
-      }
-    },
-    "picture": "samuel-valls.webp"
+      "S21": { "title": "Community Manager", "tier": 22 },
```

Comment:
I suggest to actually switch the tier from "22" to "23".

"Nodebuds Officer": { "id": 22, "index": 1200 },
"ICC Representative": { "id": 23, "index": 1250 }